chromatic/rainbows/actions/inject_noise.py

```diff
@@ -2,19 +2,23 @@
 
 __all__ = ["inject_noise"]
 
-
+# Molly! add a `number_of_photons=None` keyword argument to the function call.
+# By saying, `=None`, it will default to None.
 def inject_noise(self, signal_to_noise=100):
```

Comment:
Hi @mone0982 ! The changes you'll need to make will all be inside this `inject_noise` method. Probably the first thing you should do is confirm that that you're on this `add-photon-noise-option` branch, make an edit to the function, and then try to run it to make sure it's using your modified code. For example, try adding a silly print statement inside the function and then from a notebook try to run 
```python
from chromatic import * 
s = SimulatedRainbow().inject_noise()
```
and hopefully you'll see whatever you told it to print. That'd confirm you're working on the code in the right place! Once that's working, you can modify the function more seriously and start testing out your changes!